### PR TITLE
9826: Datatables action dropdown hidden on last row

### PR DESCRIPTION
**Pull request by Battula**
|Field|Value|
|-----|-----|
| Redmine Ticket Number | [9826](https://devops.dchbx.org/redmine/issues/9826) |
| App-Dev Road Map # | n/a |
| Start Date | 08/12/2016 |
| Due Date | n/a  |
| App-Dev Priority | n/a |
| Development Story Points | n/a |
| Peer Review Story Points | n/a |
| Ticket Author |  Anonymous |

### CODE REVIEW

**Comment by davidplappert** [comment]
@Battula it seems to keep assigning this pull request to more than one Redmine ticket. That is why the name and tags keep changing. There should only be 1 branch per pull request per ticket per dev.

I understand that this is already done, but moving forward we need to follow this pattern. Thanks!

Thanks!

File: features/step_definitions/broker_employee_quote_steps.rb [hunk 1]
```diff
@@ -186,6 +186,7 @@ def broker_agency(*traits)
 end
 
 Then(/^the broker sees that the Quote is published$/) do
+  sleep 1
```
Comment (Kevin-Wei) [inline features/step_definitions/broker_employee_quote_steps.rb]: @jfinucane nooooooooooooooooooooooo

**Comment by Battula** [comment]
@davidplappert Earlier, as per @Kevin-Wei, we have created only one branch/one PR for all the related/child tickets. Kevin asked not to created multiple branches if they're related/child tickets. Moving forward, will follow the new pattern. Thanks!

**Comment by Kevin-Wei** [comment]
addressed via https://github.com/dchbx/enroll/pull/482

**Comment by Kevin-Wei** [comment]
addressed via #856

**Comment by Kevin-Wei** [comment]
addresesd via https://github.com/dchbx/enroll/pull/856

**Pull request closed without merging**